install.py

```diff
@@ -62,7 +62,8 @@
     with open('db/initial/tvshows.json', encoding='utf-8', errors='ignore') as f:
         data = json.load(f)
         try:
-            engine.execute(TvShow.__table__.insert(), data)
+            for row in data:
```

Comment:
I made this tweak because the single transaction was talking too much time (and perhaps memory) to complete.